### PR TITLE
(re)created qa document for cops setup

### PR DESCRIPTION
**Pull request by omehes**
added qa document, qa section in index.rst and a term into glossary.rst
Note, that the qa doc will be updated to localise the setup (as per Brenda's work on this)

### CODE REVIEW

**Comment by omehes** [comment]
@m1yag1 I recreated PR #28 (https://github.com/openstax/output-producer-service/pull/28) as I used incorrect branch. I reverted that commit and created my own branch and added the qa doc there.

File: docs/qa/setup_local_qa_cops_test.rst [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+========================================================
+
+Objective
+---------
+
+Get familiar with Concourse by setting up the bakery-pipeline locally.
+This is the pipeline which is used under the hood for `Content Output Producer (COPS) <https://cops.openstax.org/>`_
+
+Retrieve Project & Config File
+------------------------------
```
Comment (m1yag1) [inline docs/qa/setup_local_qa_cops_test.rst]: These subheadings below the title should use `=============`.

File: docs/qa/setup_local_qa_cops_test.rst [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+
+.. code-block:: bash
+
+   $ ls
+
+You should see pipeline.yml listed. This file is used to configure the pipeline.
+
+Setup local concourse server
+----------------------------
+
```
Comment (m1yag1) [inline docs/qa/setup_local_qa_cops_test.rst]: Might want to include a note that these steps assume someone is using OSX

File: docs/qa/setup_local_qa_cops_test.rst [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+0. Prerequisites:
+
+- `Docker <https://www.docker.com/>`_
+
+- `fly CLI - Concourse command line tool <https://concourse-ci.org/fly.html>`_
+
+- `wget <https://www.gnu.org/software/wget/>`_
```
Comment (m1yag1) [inline docs/qa/setup_local_qa_cops_test.rst]: if assuming OSX is the case you might want to suggest installing wget via homebrew.

`brew install wget`

File: docs/qa/setup_local_qa_cops_test.rst [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+   `localhost:8080 <localhost:8080>`_
+
+Setup pipeline with local Concourse and pipeline.yml
+----------------------------------------------------
+
+1. Target the Concourse UI, so you are able to set the pipelines to UI:
+
+.. code-block:: bash
+
+   $ fly -t local-pipeline-stuff login -c http://localhost:8080 -u test -p test
```
Comment (m1yag1) [inline docs/qa/setup_local_qa_cops_test.rst]: -t is for target which would be localhost. `local-pipeline-stuff` might be better renamed to `local`, `dev-local`, or something other than "local-pipeline-stuff".

File: docs/qa/setup_local_qa_cops_test.rst [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+for the pipeline to grab.
+
+Setting up local pipeline to monitor jobs on production cops (temporary solution)
+---------------------------------------------------------------------------------
+
+1. in a terminal, run:
+
+.. code-block:: bash
+
+   cd /Users/om9/Documents/Projects/concourse-pipelines/bakery
```
Comment (m1yag1) [inline docs/qa/setup_local_qa_cops_test.rst]: might want to substitute some template text to let the user know to replace `/Users/om9/Documents/Projects/concourse-pipelines/bakery` with their own path on their local machine ex. `/path/to/bakery/repo` you can include yours as an example. if you want. Seems like you need to be more clear that you need the concourse-pipelines repo. Maybe a step where you clone it down if you need to?

File: docs/qa/setup_local_qa_cops_test.rst [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+   #     put: output-producer-updater
+   #     params:
+   #       id: output-producer-queued/id
+   #       status_id: "4" # Failed
+
+5. run:
+
+.. code-block:: bash
+
+   fly -t otto-pipeline-stuff set-pipeline -p pdf-producer -c pipeline.yml
```
Comment (m1yag1) [inline docs/qa/setup_local_qa_cops_test.rst]: rename `otto-pipeline-stuff`

Comment (omehes) [reply docs/qa/setup_local_qa_cops_test.rst]: ups... didn't see that... done!

File: docs/qa/setup_local_qa_cops_test.rst (hunk 5 above)
Comment (omehes) [reply docs/qa/setup_local_qa_cops_test.rst]: i changed this to cd into the cloned repo. Cloning of the repo is one of the first steps at the top

File: docs/qa/setup_local_qa_cops_test.rst (hunk 4 above)
Comment (omehes) [reply docs/qa/setup_local_qa_cops_test.rst]: this is an example name by Brenda. I changed it to local and noted that it can be named by the dev/qa

**Pull request merged**